sema/commons/j2/j2_functions.py

```
@@ -120,13 +161,17 @@ def xsd_format_string(content, quote, suffix):
     "xsd:string": xsd_format_string,
+    "xsd:gyear": xsd_format_gyear,
+    "year": xsd_format_gyear,
+    "yyyy": xsd_format_gyear,
+    "xsd:gyearmonth": xsd_format_gmonthyear,
+    "year-month": xsd_format_gmonthyear,
+    "yyyy-mm": xsd_format_gmonthyear,
 }
 
 
-def xsd_format(content, type_name: str, quote: str = "'"):
+def xsd_format(content, type_name: str, quote: str = "'") -> str:
```

Ruff (0.8.2)

Missing type annotation for function argument `content`

(ANN001)